apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@nestjs/common';
+import { IntegrationEntity, IntegrationRepository } from '@novu/dal';
+
+import { CreateNovuIntegrationsCommand } from './create-novu-integrations.command';
+import { CreateIntegration } from '../create-integration/create-integration.usecase';
+import { CreateIntegrationCommand } from '../create-integration/create-integration.command';
+import { GetNovuIntegration } from '@novu/application-generic';
+
+@Injectable()
+export class CreateNovuIntegrations {
+  constructor(private createIntegration: CreateIntegration, private integrationRepository: IntegrationRepository) {}
+
+  async execute(command: CreateNovuIntegrationsCommand): Promise<IntegrationEntity[]> {
+    const integrations: IntegrationEntity[] = [];
+
+    if (
+      !process.env.NOVU_EMAIL_INTEGRATION_API_KEY ||
+      !process.env.NOVU_SMS_INTEGRATION_ACCOUNT_SID ||
+      !process.env.NOVU_SMS_INTEGRATION_TOKEN ||
+      !process.env.NOVU_SMS_INTEGRATION_SENDER
+    ) {
+      return [];
+    }
+
+    const emailProvider = GetNovuIntegration.createNovuEmailIntegration(null, command.environmentId, true);
+    let count = await this.integrationRepository.count({
```

Comment:
To avoid ambiguity, would suggest renaming this to `emailProviderCount` and `smsProviderCount` below, so we won't have to override the count variables